### ISSUE TITLE
Peer Review

### ISSUE THREAD

**Issue by anissadallmann** [cmh342/4741Project#3]
This project seeks to given information on a company, will it go bankrupt or not. The authors plan to use data on around 7000 different Taiwanese companies' bankruptcies collected by the Taiwan Economic Journal between 1999 and 2009. Using features derived from this dataset as well as features further engineered, the authors plan to try out several different types of classifiers to gain insight into which financial indicators best predict bankruptcy and eventually make a prediction.

What I like about this project:
1.  I think that this is a very important and useful question to answer in the financial world. A model that could accurately predict bankruptcies would be very interesting to investors or companies in making financial decisions since a company going bankrupt would have major financial repercussions for them.
2. I like the discussion considering false positives and false negatives and the weights and relative importance placed on both of them.. I think it will be something very useful to consider when deciding on which loss function to use, especially given how many possibilities there are, I think it is good that you are already thinking about them.
3. I think that, as you mentioned, it is a good idea to try out different types of classification models (SVDs, linear regression, and decision trees) since one might fit the question better than others. 
4. I also think it's great that there is so much information available on the companies (96 features) in the dataset, it will definitely give you a lot of options when engineering features.

What I would add:

1.  I think the question of "will a company go bankrupt or not" is still a little vague. I think there is a big difference in predicting whether a company will go bankrupt within the next few months vs. within 10 years or so. It may be better to refine the question a bit since depending of which timeframe you want to predict in, the features could look very different (and different features may be more useful in one time span than another).
2.  Are you planning to generalize the model to countries other than Taiwan? If yes, it may be good to look for other datasets for other countries as well, since there may be other different information available on companies based on which country they are based in. Otherwise you may be able to do some research about which of the features in your Taiwan dataset are available for other countries' companies and try to only use those for your model.
3. Adding onto the previous point, the dataset seems a little small (7000 companies), especially when 20% should be reserved for your test set. It might help to find some additional data sets for better results. If not, you could always consider using cross-validation with a large number of splits for your train/validation sets.
4. Since this is almost a type of timeseries-type prediction, I think it would be good to consider how you want to split up your test and training sets as well as how much data (i.e. how far back) you want to go when engineering features for each company. I think this also relates back to my first point about deciding how far out in the future do you want to predict a companies' bankruptcy.
